pom.xml

```diff
@@ -212,6 +303,11 @@ THE SOFTWARE.
       <artifactId>jmh-generator-annprocess</artifactId>
       <version>${jmh.version}</version>
     </dependency>
+    <dependency>
+      <groupId>jakarta.servlet</groupId>
+      <artifactId>jakarta.servlet-api</artifactId>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
Matching the plugin parent POM. Also requires the same addition to the Jenkins core test suite, as in https://github.com/jenkinsci/jenkins/pull/10554.